src/shared/tpm2-util.h

```diff
@@ -452,13 +452,16 @@ typedef struct {
 typedef enum Tpm2Support {
         /* NOTE! The systemd-creds tool returns these flags 1:1 as exit status. Hence these flags are pretty
          * much ABI! Hence, be extra careful when changing/extending these definitions. */
```

Comment:
Hmm, such change is a bit intrusive and overall not really useful for other users of tpm2_support(). Maybe introduce a separate enum for this, and avoid changing the retval?

Reply:
Currently, TPM2_SUPPORT_LIBRARIES is used only in has-tpm2 command and tpm2-generator for logging.
So, hence, splitting `TPM2_SUPPORT_LIBRARIES` into three does not break anything.
Also, to set TPM2_SUPPORT_LIBRARIES flag, anyway all three libraries need to be successfully loaded. Hence, I do not think the the change is intrusive. Of course, the fine grained flags are used by has-tpm2 flag, but I do not think it is not a big matter.

Reply:
To be more specific, I simply want to avoid hardcoding the lib names in a enum :/

> Currently, TPM2_SUPPORT_LIBRARIES is used only in has-tpm2 command and tpm2-generator for logging.

No? As stated, has-tpm2 also uses the inverted mask as exit status.

Reply:
> No? As stated, has-tpm2 also uses the inverted mask as exit status.

Ugh... Indeed. But, that's not documented 😝 But, ok, I will make the exit status unchanged.